### ISSUE TITLE
UV index has incorrect maximum value of 10

### ISSUE THREAD

**Issue by salmundo11** [naofireblade/homebridge-weather-plus#201]
```
[homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.

 [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.

 [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.

```
Node.js: 14.16.1
NPM: 7.11.2
Homebridge: 1.3.4
OS: MacOS 10.13.6

**Comment by benchristian88** [comment]
same here.
1/29/2022, 12:32:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 12:37:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 12:42:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 12:47:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 12:52:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 12:57:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:02:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:07:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:12:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:17:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:22:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:27:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:32:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:37:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:42:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:47:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:52:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 1:57:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 2:02:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 2:07:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 2:12:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 2:17:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.
[1/29/2022, 2:22:16 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'UV Index': characteristic was supplied illegal value: number 11 exceeded maximum of 10. See https://git.io/JtMGR for more info.

**Comment by Zerosignal84** [comment]
I am also having the same issue

**Comment by naofireblade** [comment]
Fixed in v3.2.7
